config.js

```diff
@@ -68,14 +68,22 @@ Config.prototype._seed = function _seed(seed) {
 
     // Logger configs
     seedOrDefault('defaultLogLevel', LoggingLevels.info);
-    seedOrDefault('dampingLogLevel', LoggingLevels.warn);
+    seedOrDefault('dampingLogLevel', LoggingLevels.error);
     seedOrDefault('gossipLogLevel', LoggingLevels.off);
     seedOrDefault('joinLogLevel', LoggingLevels.warn);
+    seedOrDefault('suspicionLogLevel', LoggingLevels.off);
```

Comment:
Should all logging levels be at error, at a minimum? Even if they currently don't emit errors, this seems like the best default.